pkg/sloop/webserver/resourcehandler.go

```diff
@@ -83,10 +84,10 @@ func resourceHandler(resLinks []ResourceLinkTemplate) http.HandlerFunc {
 		queryEnd := d.ClickTime.Add(d.PlusMinusTime).Unix()
 
 		dataParams := fmt.Sprintf("?query=%v&namespace=%v&start_time=%v&end_time=%v&kind=%v&name=%v", "GetEventData", d.Namespace, queryStart, queryEnd, d.Kind, d.Name)
-		d.EventsUrl = "/data" + dataParams
+		d.EventsUrl = path.Join("/", currentContext, "data" + dataParams)
```

Comment:
Are `dataParams` actually part of the `path`? If everything past the `?` is considered params for the call, then we should move the `+ dataParams` outside of the call to `path.Join`. If you think of the entire URL as the path, then it is fine where it is.

Reply:
yeah I would like to keep it here for clarity purposes